rllib/core/rl_module/rl_module.py

```diff
@@ -533,7 +533,11 @@ def _from_metadata_file(cls, metadata_path: Union[str, pathlib.Path]) -> "RLModu
         module = module_spec.build()
         return module
 
-    def save_to_checkpoint(self, checkpoint_dir_path: str) -> None:
+    def _weights_relative_path(self) -> pathlib.Path:
+        """The relative path to the weights in the checkpoint."""
```

Comment:
Can we get a more elaborate docstring. This is some important concept that we missed in our review and it's important to make it super clear why this was added.

Reply:
+1

Reply:
ok I actually just changed the name of the function, and removed the responsibility of making this a path and instead just made it a function called  `_module_state_file_name` which should be a lot easier to understand

Reply:
I'm guessing you have not pushed it yet?
+1 to these changes, name is better.

Reply:
wait no I pushed it, but the view we're looking at is outdated
<img width="930" alt="image" src="https://user-images.githubusercontent.com/38871737/224842612-ea00b697-e8fd-4277-aafb-d50f7a76e4b8.png">

Reply:
Ah thanks, sorry.